### DISCUSSION TITLE
[Community] Plugin requests

### DISCUSSION THREAD

**Discussion by th-ch** [pear-devs/pear-desktop#93] [Ideas]
Opening this discussion to know what new plugins are wanted in the community! 

Feel free to answer here your plugin ideas! (e.g. Discord integration)

**Comment by LatvianBHood** [comment]
A UI editor would be super cool. Maybe a resolution option for video's and a caption toggle.

**Reply by Zippu-4249** [reply]
Yeah an Ui editor is a great addition I hope they add it

**Reply by chamodyoshi123** [reply]
What do you think abut this UI https://github.com/chamodyoshi123/Liquid-Glass-theme-by-Chamod-Yoshitha

**Comment by wyvernone** [comment]
Downloader plugin not working with:
error: no video id found:
https://music.youtube.com/playlist?list=.....

When will this plugin work with youtube music premium? Would love to have this feature to export an mp3 of a song/playlist, etc.

**Reply by LatvianBHood** [reply]
For me it just infinately tries to download and fails

**Comment by mavzolej** [comment]
I'm wondering how the download music quality is determined? I downloaded a bunch of different songs and they all were 64kbit/s. Would be nice to enable making the quality higher.

**Comment by VanGeko** [comment]
im missing a plugin for
*Skip YouTube Music Dislikes
*

an plugin, that sees Songs/videos i have disliked and skip these songs/videos.

**Comment by prismagod** [comment]
It will be great to add some functionality for picture-in-picture mod, like click-through and remove borders.

**Reply by Zo-Bro-23** [reply]
What do you mean by "click-through"?

And currently the PiP plugin doesn't show any borders, does it (I might be wrong)?

**Reply by prismagod** [reply]
Sorry for misunderstanding, I mean the header with the name of app and window buttons. 
![image](https://user-images.githubusercontent.com/97229646/214848130-d86671f7-9d30-4d49-8cb2-0838fb5c1a0b.png)

And click-through I mean that I can't manipulate with the window by mouse. To click on window again I will need to press some global hotkeys or toggle "click-through" setting in app menu (by right-clicking on the app icon in tray). 

It will helps alot! I love to play some MMO's and watch clips on youtube. I put this window on the chat panel and farm, so I don't want to click on the youtube-music window while playing a game.

**Reply by Araxeus** [reply]
> Remove Titlebar

This can probably be easily added as a feature if you use `in-app-menu`, 

Without `in-app-menu` – that requires recreating the window with [`frame` set to `false`](https://www.electronjs.org/docs/latest/tutorial/window-customization#create-frameless-windows), which means the page would reload… not ideal

**Comment by MisterCHOUN** [comment]
Hello
I would like to request a "sorting songs inside playlist" plugin if possible

Yeah Youtube allows you to sort alphabetically your list of playlists

![sorting list of playlists](https://user-images.githubusercontent.com/65115291/215263035-3231c303-bfe7-40e6-a1ed-0ed4f26f003d.PNG)

But not what is INSIDE the playlist 

![Can't sort alphabeticaly songs inside playlist](https://user-images.githubusercontent.com/65115291/215263126-ad9d15f6-ccd7-4e88-9130-cb516110cbae.PNG)

(I hope I'm not wrong about this one, but ) you cannot sort alaphabetically songs inside your playlist (-__-) ...

Thank you in advance

**Comment by filispeen** [comment]
Hi @th-ch , how do you like the idea of ​​adding an **equalizer**?

**Comment by th-ch** [comment]
@filispeen Sounds like a good idea, feel free to create a plugin and open a PR!

**Comment by anisharbi** [comment]
hey i would like to request an inside vpn or proxy or a bypass for countries that doesnt have youtube music yet
![image](https://github.com/th-ch/youtube-music/assets/60405107/84b72e78-17f1-47b7-90c9-55f64de3fa21)

**Reply by youttiao** [reply]
seems it do not include system proxy settings
found in advanced settings

**Comment by Destroy666x** [comment]
A way to search & request songs through CLI or some sort of API would be awesome. E.g.:
```
ytmusic queue "Rammstein Ohne Dich"
```

**Reply by Tyr808** [reply]
Came here to request the same kind of idea myself. Not sure if this is geared towards streaming, but if so an idea I had for trying to do this myself was to get input by loading a local text file since things like SAMMI and streamer.bot can easy write to one with input from chat. I believe that all of the core functionality is there to take that input, run it as a search and add the top result to the queue, but unfortunately I was in over my head when it came to actually trying to write the plugin itself, haha.

**Reply by ArjixWasTaken** [reply]
A dedicated issue for this exists, #1581

**Reply by Destroy666x** [reply]
Yeah, streaming, to integrate with e.g. Advanced Scene Switcher. I started working on API rather than CLI as it's seemingly much easier to achieve with Electron, for now basic version, but song queue/requests will be next feature to add to that after I finish soon™ @Tyr808

**Comment by redmineman** [comment]
an add on to the discord rich presence 'listen along' feature where at the end of the url, it has &t=(timestamp in song in seconds) so it plays at the part that the user is up to
![image](https://github.com/th-ch/youtube-music/assets/145813337/c5b05c6e-c88d-41e4-b650-4f37c1d45a66)
![image](https://github.com/th-ch/youtube-music/assets/145813337/aee2523c-42b4-4723-845c-3d0c7df46771)

**Reply by Destroy666x** [reply]
This is not really a plugin request but more of something that should be an issue IMO. Discord plugin already exists anddoesn't need to be requested and the functionality will be part of it.

**Comment by Rocker2234** [comment]
Add Rainmeter and MacroDeck support with all the functions. A browser extension [WebNowPlaying-Redux](https://github.com/keifufu/WebNowPlaying) which helps in getting the data from the player to the other apps, also does basic controls. This would be really helpful.

**Reply by Khaivn1308** [reply]
OMGG, i need this so badly

**Reply by Destroy666x** [reply]
Rainmeter support already exists with HTTP support, it just needs to be combined with a widget properly.

**Reply by Khaivn1308** [reply]
i found a way to make rainmeter work with th-ch app
[https://forum.rainmeter.net/viewtopic.php?t=39164](url)

**Reply by Khaivn1308** [reply]
> Rainmeter support already exists with HTTP support, it just needs to be combined with a widget properly.

MediaPlayer.dll will pull song info from windows itself and feed that into rainmeter
[https://forum.rainmeter.net/viewtopic.php?t=39164](url)

**Comment by clcocd** [comment]
I think it would be really convenient to have a plug-in for Stream Deck

**Reply by ArjixWasTaken** [reply]
And what would a plugin for stream deck do?

**Reply by clcocd** [reply]
You can check information about the songs you're playing and control playback/pause and the next song/previous song

**Reply by ArjixWasTaken** [reply]
~~What you are asking for is already a thing, in fact it is supported on all platforms.~~
~~I think you meant to ask for a decky-loader plugin, which would be out of scope for this project.~~

~~Maybe this plugin does the job for you?
https://github.com/Philip-Scott/media-controls-plugin~~

**Reply by clcocd** [reply]
It's not SteamOS. It's Stream Deck

**Reply by ArjixWasTaken** [reply]
Ok, I misread 

Sorry for my deleted message, I apologize

**Reply by ArjixWasTaken** [reply]
Dedicated discussion for StreamDeck [here](https://github.com/th-ch/youtube-music/discussions/1650).

**Comment by bykbtzr** [comment]
A plugin to enhance playlist editing would be great.
Things like these would be very helpful:
- improve drag and drop performance (perhaps hiding track thumbnails)
- multiple selection for mass actions (remove, move to top, etc)
- range selection (shift + click)

**Reply by Zippu-4249** [reply]
Yeah improve the perfomance of the app and yeah all these feature would be amazing also add a bigger volume slider , youtube music links directly redirects to the app + you should be able to open playlist link

**Comment by Roma2047-png** [comment]
Hi! I try to download music with the download plugin and it gives me a message on the screen of 'RageError: WebAssebly.Memory(): could note allocate memory in' and then some numbers. Please help!. 🙌🙌🙌

**Comment by TheGeekno72** [comment]
Hi there, I'm trying to use 6KLABS' Amuse widget for OBS and in order to connect it to YT Music, the only compatible app is this : https://github.com/ytmdesktop/ytmdesktop
Unfortunately they do not have any adblocker plugin fitted in and when asked about it, they don't support putting adblockers in their software and don't plan on putting adblocking plugins ~~(yet ?)~~ **ever** but I'm not going to give those Google f*cks a single cent anymore (16€ for YT Premium ? what am I, made of money ?) and I don't do code so I haven't the faintest idea how to solve this, Proton's NetShield and AdGuard's desktop app don't catch the ads...
Do you think it would be possible to integrate the Remote Control feature of YTMDA in a way that allows 6KLABS' Amuse to read the content played so I can use YT Music in my streams and still have adblocking (on top of all the other plugins you already crammed in there) ?
Hopefully that's a doable task, Amuse is the best tool I could find that does what it do with practically no competition but idk why they only got one option available for YT Music data pulling...

Thank you for your time and consideration !

**Reply by ArjixWasTaken** [reply]
I doubt that we are going to implement a compatibility layer for ytmd's remote control.
But there is a similar [plugin](https://github.com/th-ch/youtube-music/pull/1909) already in the workings.

Once that is finalized, 6klabs could choose to integrate with it.

**Comment by FunWarry** [comment]
Just like the YouTube playlists already existing on the YouTube platform, we have the possibility with "check boxes" to decide in which playlist we can put the videos (see illustration :)

![image (1).png](https://github.com/user-attachments/assets/ac56a51c-8b95-4202-bc78-dc6ebc0cb3e2)

As you can see, videos can be put into multiple playlists in one go.

We also have the possibility, by clicking on a video already added in a playlist, to see in which of all our playlists the video has been saved. (see illustration :)

![image (2).png](https://github.com/user-attachments/assets/a146a7ae-4968-4163-ade7-e1783727bb59)

What I'm getting at is whether you were planning to put this system on YT music one day.
Because for the moment, we neither have the possibility to add a song to several playlists at once, but we also cannot see if a song is in one or several playlists only by going to the add playlists menu, which would be extremely practical.
(see illustration :)


![image (3).png](https://github.com/user-attachments/assets/bff2fed7-59e7-4865-b22e-9c14ef9bfefa)

**Reply by FunWarry** [reply]
up https://github.com/th-ch/youtube-music/discussions/728#discussion-4093805

**Reply by ciniseb** [reply]
This would be amazing instead of always going to Youtube

**Comment by FunWarry** [comment]
A search bar only in our playlist, and in playlist, not only in the entire YT music

**Comment by Zippu-4249** [comment]
Fix the download plugin and polish it [ it gives so many crashes , while downloading an entire playlist ] make the app polished
check my 2 discussons I posted but no one saw them which is sad for me .

**Comment by Zippu-4249** [comment]
Make the windows hover thing ui [ like when its on task b and you hover on it to play/pause , next , previous the blue color is so bad if possible make it ambient so it changes color depending on the content's coler if thats not possbile just make it gray or smth. And I hope you guyz make an Youtube app soon

**Comment by Helxcate** [comment]
If possible, a plugin or option that enables window transparency (much like Better Discord and Vencord's options) to allow for CSS themes to make use of transparent backgrounds.

**Comment by nachevm** [comment]
I really hate that whenever I click play on any song the video playlist pops up and have to click to go back the search or wherever. Just play the music in the bar below like Spotify

**Comment by nalexeyev** [comment]
VST Plugin support (to use e.g. with [morphit by toneboosters](https://www.toneboosters.com/tb_morphit_v1.html) for calibrated headphones sound)

**Comment by Andi9910** [comment]
romaji lyrics fix when?

**Reply by ArjixWasTaken** [reply]
What do you mean by "fix"?

**Reply by Andi9910** [reply]
![image](https://github.com/user-attachments/assets/9f79d6d6-5a3e-4f7e-a4be-05a56b40386f)

it just straight up doesnt work

**Reply by ArjixWasTaken** [reply]
lyrics genius itself doesn't work at all...
use the new synced-lyrics plugin, but romanization is not implemented for it yet

**Reply by Andi9910** [reply]
yeah... all i want is romanized lyrics

**Reply by ArjixWasTaken** [reply]
I've started work on this, if you have constructive feedback and ideas, you can share them as comments under that [PR](https://github.com/th-ch/youtube-music/pull/2790) or the related [discussion](https://github.com/th-ch/youtube-music/discussions/2382).

**Reply by Andi9910** [reply]
Nice nice, i appreciate it, but im just a user that knows nothing about coding lol
Hope you the best of luck

**Comment by Bagaco3763** [comment]
please add local files support

**Comment by prohunterallen** [comment]
Is any possible to add support plugin with 'Sleeve 2' on macos
[Sleeve 2](https://replay.software/help/sleeve/developers)

**Reply by ArjixWasTaken** [reply]
Adding support for applescript looks hard, especially since google doesn't yield great results.

I'd suggest someone makes an external wrapper that uses the http-api plugin

**Comment by hedy7277** [comment]
hey i have this really crappy $150 goodwill laptop, could you please add a volume increase extension like this one? thank you
![Screenshot 2025-01-19 at 11-21-13 600% Sound Volume – Get this Extension for 🦊 Firefox (en-US)](https://github.com/user-attachments/assets/8abc0d4f-387a-452f-94e1-52de4c4b3a15)

**Comment by JChungus07** [comment]
How about a scrollbar styling plugin?

**Reply by ArjixWasTaken** [reply]
~~how about you write some CSS?~~
Jokes aside, you can already achieve that by writing CSS, if you don't know CSS, then use ChatGPT.

**Comment by LockeCPM4** [comment]
I would love to see a plugin that does an overlay for when I am using this application while gaming. Maybe just have it briefly pop up when the next song is playing showing the song title or something

**Comment by ArFrS** [comment]
I think it would be cool if there were a feature that could display the lyrics in a Picture-in-Picture like window on the screen.

**Reply by ArjixWasTaken** [reply]
already has an issue for it, although the language used "floating lyrics" is a bit ambiguous
https://github.com/th-ch/youtube-music/issues/650

**Comment by Sebixer23** [comment]
I would love to be able to like the song currently playing or dislike it just from the taskbar icon (right click -> dislike/like song) instead of having to show window and then dislike/like.

**Comment by Reaper8f** [comment]
### Remote Control Functionality like Spotify
I propose integrating remote control capabilities into this app. I don't know what technical skills are needed for this integration, but a plugin would be awesome. It can work over Bluetooth or via a device connected to the same network. A simple app on Android/iOS can be used to control the music or can be used via a web browser from a mobile phone. 👍

**Reply by ArjixWasTaken** [reply]
you *could* use [KDE Connect](https://userbase.kde.org/KDEConnect#Multimedia_Control_Receiver) to connect the media playing on your computer (even firefox/chrome/whatever)

**Reply by Reaper8f** [reply]
nah aint working,

**Reply by Lockszmith-GH** [reply]
Check out https://www.music-assistant.io, it's not just a simple app install - but that replaces all remote controlled services, regardless of the speaker you have.

**Reply by JellyBrick** [reply]
We already have this feature.

Please enable the `api-server` plugin and use this application: https://github.com/franz-dc/youtube-music-remote

**Reply by Reaper8f** [reply]
> We already have this feature.
> 
> Please enable the `api-server` plugin and use this application: https://github.com/franz-dc/youtube-music-remote

Well the web browser type won't work as i can't find insecure thing in site setting on chrome nor ios or android. The android apk works, only down low is i can't control volume with my device volume buttons - it be easy is this was so. Oh well this very limited i can't add/change/remove music. Only feature is next/previous song and volume.

**Comment by infinityswift** [comment]
Is there something for just voice?  Youtube Music stop music or play xxx. 
Maybe something simple for now just start, stop, next, back.

**Comment by radical4ever** [comment]
would love to have a plugin (or css) that allows to have a transparent top bar like on the mac os app on windows. (i leave the image for better understanding)
![screenshot](https://github.com/user-attachments/assets/b6c6a5b8-759c-4881-a9e8-ddba812cfdce)

**Reply by ChanglaManus** [reply]
I am on Kubuntu 24.04 KDE 5.27 with MacOS theme installed, I tried to change settings to get this look but due to KDE UI I got a Grey Titlebar on top with MacOS buttons on the left side, the Global menu is well supported though.
I too would like if there are any CSS themes for this app that would allow us to make it look like the screenshot.

**Comment by TheWildPlant** [comment]
We need a plugin?(Don't know what to call it) For Android phones

**Reply by ArjixWasTaken** [reply]
Use revanced.

**Reply by Destroy666x** [reply]
Grayjay

**Reply by ArjixWasTaken** [reply]
Since when is grayjay for music?

**Reply by Destroy666x** [reply]
Since always. It's for everything, just like Vanced.

**Reply by ArjixWasTaken** [reply]
You realize ReVanced has a mod for the official youtube music app on mobile, and you are suggesting a video streaming app?

**Reply by Destroy666x** [reply]
Yeah, that's indeed my recommendation. It has all the features needed to stream YT music playlists without breaks, video, etc. It also has plugins for e.g. SoundCloud or Spotify. Not like official YouTube Music is very feature-rich in the 1st place... Nowhere close to this 3rd party desktop app.

**Reply by ArjixWasTaken** [reply]
I might as well recommend this here then
https://github.com/brahmkshatriya/echo

an extension-based music client, that can play yt/spotify/deezer/whatever songs, and many other stuff

**Comment by XketsuroX** [comment]
A suggestion that may improve quality of life? A plugin that can display lyrics always on top as an overlay. It will be comfortable if I can check the lyrics while working.

**Reply by ArjixWasTaken** [reply]
duplicate suggestion, many tickets have been made for it

**Reply by XketsuroX** [reply]
> duplicate suggestion, many tickets have been made for it

Oh, I'm sorry for that.

**Reply by ArjixWasTaken** [reply]
nah don't be

**Reply by Lockszmith-GH** [reply]
If you are using Windows - there is a tool in PowerToys called [Crop and Lock](https://learn.microsoft.com/en-us/windows/powertoys/crop-and-lock).

**Reply by JellyBrick** [reply]
See https://github.com/organization/alspotron/blob/master/README-i18n/en.md

**Reply by XketsuroX** [reply]
alspotron seems great, I'll try that out later

**Reply by XketsuroX** [reply]
> If you are using Windows - there is a tool in PowerToys called [Crop and Lock](https://learn.microsoft.com/en-us/windows/powertoys/crop-and-lock).

yeah, I'm using that now, but just looking for a better one? like with cursor pass-through and transparent background

**Comment by Wyth** [comment]
For the love of all that is holy, the single thing I need (that drives me crazy that Google doesn't just do itself (also, no pun intended)) is a way to turn off singles from suggestions and new releases. Please let me have a way of seeing things that are at least EPs and Full Albums instead of have to wade through all the singles.

**Comment by NotSkele** [comment]
I don't think I've seen anyone ask but a plugin to disable albums playing videos? It hinders the listening experience as that feature usually chooses live or cut versions of songs and rarely plays the actual album song version for more popular albums. I haven't seen any option in settings to disable it, would be a very convenient feature. Mb if I missed this topic, thanks in advance!

**Reply by livingperson** [reply]
Yeah this huge. Often some random stylised video version of the song plays, with random talking and other non-music sound.  Ruins music listening experience entirely.

**Comment by ikarot** [comment]
I recently discovered this program and was wondering if this has been requested before, and if not, is a Songify plugin possible?

**Reply by ArjixWasTaken** [reply]
what does that entail?

**Reply by ikarot** [reply]
I currently use Songify so that people in the Twitch chat can request music with a command like !sr song-artist, which allows them to change the music on my Spotify and I was wondering if the same thing could be replicated by interacting between Songify and YT Music.

**Reply by ArjixWasTaken** [reply]
Like [[Feature Request]: Twitch functionalities](https://github.com/th-ch/youtube-music/issues/2043) ?
https://extensions.streamer.bot/t/song-request-system-youtube-music/2743

**Reply by ikarot** [reply]
Yes, something similar to what you show with Streamerbot, I send an image of the commands that Songify already has that in this case allow you to add songs and other things with Spotify

![Screenshot_23](https://github.com/user-attachments/assets/02c9abe6-a684-4143-beb5-49f8258b842b)

**Reply by ArjixWasTaken** [reply]
yeah that sounds like a thing songify has to do, so why not make an issue on their project?

they already support a similar project (YTMD), they could add support for this project as well

https://github.com/songify-rocks/Songify/tree/master/Songify%20Slim/Util/Songify/YTMDesktop

**Reply by ikarot** [reply]
Oooh, I see they've released a new version compatible with YT Music. I'll ask them if they can add the project. Thank you very much for your attention and sorry for the inconvenience.

**Comment by lexova** [comment]
I hope, the Transpose plugin will be here soon. ^_^

**Comment by phoenixv97** [comment]
I'm not sure if anyone has suggested. Is it possible to make a plugin for playlist folders? I would love organization because I struggle to find my playlists.
And maybe perhaps a dyslexia font plugin?

**Comment by ejordan9** [comment]


**Reply by ArjixWasTaken** [reply]
![image](https://github.com/user-attachments/assets/09f6774d-0af4-4d65-ab19-ec3651a47d19)

**Reply by ArjixWasTaken** [reply]
Currently, no.

**Comment by Paddywan61** [comment]
I would kill (and donate what I can afford) for a shuffle album or even just the same random album function from this extension (https://chromewebstore.google.com/detail/random-album-for-youtube/obkkkldnmaoahhpkhomdmdpjldcpihph) .

**Reply by livingperson** [reply]
That's a great suggestion.

**Comment by NearRaindrop116** [comment]
Absolutley no idea if it would be possible but maybe a semi transparent clickthrough overlay that you can put in the bottom left corner or top right for people like me who dont have a second monitor. you likley display things such as name and current time and a toggle to also show the current and next lyric line thing. i suck at wording but im sure someone understands the idea

**Comment by livingperson** [comment]
Reading, writing, and interacting with YouTube Music comments is something I really miss when I use this app. I've loved interacting with other listeners when using the iPhone app.

**Comment by Bamboooi** [comment]
> Opening this discussion to know what new plugins are wanted in the community!
> 
> Feel free to answer here your plugin ideas! (e.g. Discord integration)

Lyrics Overlay when using other programs something like this image from other music player.
<img width="992" height="186" alt="image" src="https://github.com/user-attachments/assets/ae6805a2-27eb-4244-a674-de17752b290c" />

**Comment by Huntrezz54** [comment]
Hello! Was wondering if there could be a "transpose" function to transpose the pitch of the music up or down? Something similar to the google extension would be nice!!

**Comment by SneakyZucchini** [comment]
Apologies is this isn't the best place to post this - I have tons of playlists with funky names. All I would want in life is a way to reorder playlists (Not songs in a playlist). Maybe it's already possible? I would love to just be able to drag and release to create order - maybe a custom tab?

**Comment by ClaudioAFC** [comment]
The download function often doesn't work (in the new release it's once again broken).

Would be nice to have a settings option on it to change the parameters for when it happens, or even to change the download source/settings.

**Comment by SmollClover** [comment]
A plugin that adds buttons in the bar in the bottom where the audio slider is that gives volume presets. Akin to the Spicetify extension [Volume Profiles](https://github.com/notPlancha/volume-profiles-v2/).

It would add three buttons, or maybe a variable amount defined by the user, which can be edited to set the volume to a certain percentage on the click. I often switch to different volumes when I'm wearing headphones and when I'm listening over speakers, this would allow the user to just click a single button to set the volume, for example, to be at 30%, and with another click, to be at 100%, or whatever the user defined.

**Comment by NanKillBro** [comment]
A fullscreen plugin that similar to Apple Music
<img width="1920" height="1080" alt="image" src="https://github.com/user-attachments/assets/3ca8fb24-21a9-4dab-be93-35fa8d2e6855" />

**Reply by Paulogo12345XD** [reply]
Kinda like Better Lyrics for the web version
https://chromewebstore.google.com/detail/better-lyrics-lyrics-for/effdbpeggelllpfkjppbokhmmiinhlmg?hl=en

**Comment by RiquezGHOST** [comment]
Would it be possible to create a plugin that lets the desktop app connect with a companion website or mobile app, so we can control playback remotely?

Something like Spotify Connect: being able to pause, skip tracks, change volume, and switch songs from a phone without needing the desktop app open in the foreground.
Basically, remote media control synced between PC and mobile.

**Reply by Beastlybear2017** [reply]
You can try: https://github.com/franz-dc/youtube-music-remote

Only thing to note is that on windows, I had to add a firewall rule exception for the app in order for my phone to be able to connect.

**Comment by TogoFire** [comment]
Suggestion: Pitch adjustment

Hello, I would like to request a new feature: a pitch adjustmeennt function in semitones, similar to the one available in Neutron Music Player or extensions 432convert (Chrome) and Frequency Tuner (Firefox).

There is a Chrome extension that does this perfectly: 
https://432convert.com/
https://432convert.com/chrome-extension
https://chromewebstore.google.com/detail/432hz-converter-real-time/kpcncfndlejbpgfekadgaoobdaheecbl
https://addons.mozilla.org/pt-BR/firefox/addon/frequency-tuner/

It would be a great feature for those who prefer this tuning. Thanks!

**Comment by chamodyoshi123** [comment]
Hey!  I made an automixing plugin for this app . But, I can't add it to the app .  I mean I couldn't find any plugin folder. I've created a plugin folder it also didn't worked.  By the way I also made a theme . You can download it here 👉https://github.com/chamodyoshi123/Liquid-Glass-theme-by-Chamod-Yoshitha

**Comment by dingus1293** [comment]
Hi folks. Would it be possible to include the full artist title in the Discord integration? Currently, only the first artist of the track is shown instead of all the artists.

**Comment by HarryTheHeath** [comment]
Suggestion: Album shuffling

Hi gang. Would love to see some support for Album shuffling, for those of us who like to mostly listen to one record at a time, but don't want to let old favourites get buried in to obscurity.

Someone has developed a Chrome extension for this already, too, which might be integratable: 
https://github.com/cgarst/random-album-for-youtube-music

**Comment by clarjon1** [comment]
Suggestion: Disable pausing when youtube detects playback on another device.

Sometimes I open a youtube video for a quick guide. ANd that stops my playback in the player. And that's SUPER ANNOYING!!

**Comment by gamatoto222** [comment]
Hi! Is there any way to integrate searching for lyrics by title only, or selective searching, so that cover music can still display the original song's lyrics?
<img width="1918" height="982" alt="image" src="https://github.com/user-attachments/assets/5d115be0-d5de-422b-97b3-1267cd18b09d" />

<img width="1909" height="898" alt="Screenshot 2026-08-10 183705" src="https://github.com/user-attachments/assets/80f40108-a8e4-4cc7-b8f3-9b02990ef862" />

It's playing the same song, but the 1st photo is the original and the second one is a cover song, but I want the cover song to show the same lyrics. thanks!
